indico/modules/events/timetable/client/js/utils.ts

```diff
@@ -140,3 +140,13 @@ export function getEntryColor(
 export function formatBlockTitle(sessionTitle: string, blockTitle: string) {
   return blockTitle ? `${sessionTitle}: ${blockTitle}` : sessionTitle;
 }
+
+export function shiftChildrenStartDt(entry: Entry, currentDay: moment.Moment): ChildEntry[] {
+  const deltaStartDt = moment.duration(entry.startDt.diff(currentDay));
+  const newChildren = entry['children'].map(child => ({
+    ...child,
+    startDt: moment(child.startDt).add(deltaStartDt),
+  }));
+
+  return [...newChildren];
```

Comment:
map already returns a new array:
```suggestion
  return newChildren;
```